app/core/Tracker.coffee

```diff
@@ -169,6 +169,7 @@ module.exports = class Tracker extends CocoClass
   trackSnowplow: (event, properties) =>
     return if me.isSmokeTestUser()
     return if event in ['Simulator Result', 'Started Level Load', 'Finished Level Load']
+    return if properties?.category is 'Amazon' # No slowplow schema for these yet
```

Comment:
It'll be clearer if we enumerate the events to skip in the above line, rather than keying off the event category. We would not expect giving an event the category "Amazon" to have this behavior.